app/controllers/application_controller.rb

```diff
@@ -126,4 +126,11 @@ def editor_preview?
     request.script_name.include?('preview')
   end
   helper_method :editor_preview?
+
+  def service_slug_config
+    @service_slug_config ||= ServiceConfiguration.find_by(
+      service_id: service.service_id,
+      name: 'SERVICE_SLUG'
+    )&.decrypt_value
```

Comment:
Does this need to be stored encrypted? Might be easier for us if they're not and we need to search by them. No big deal if it's hard to change though.